### ISSUE TITLE
cf-checker requires numpy <= 1.8

### ISSUE THREAD

**Issue by pelson** [cedadev/cf-checker#2]
I opened an issue on @RosalynHatcher 's CFChecker repo, but I suspect that isn't the primary cf-checker repo (and this is).

Please see https://github.com/RosalynHatcher/CFChecker/issues/1:

> The latest version of numpy has completely removed the deprecated old numeric code. CFChecker is still making use of this code, therefore cfchecker doesn't support v1.9 of numpy.

> The port should be pretty straight forward - there is only limited use of Numeric, and it looks like it is a 1-to-1 replacement with numpy's ndarray (untested).

If this is the primary repo, I don't mind submitting a PR which tries to reduce the use of ``numpy.oldnumeric.Numeric``, though not having any tests to check I'm not breaking anything makes that a little more daunting.

Cheers,

**Comment by RosalynHatcher** [comment]
Hi Paul,

Sorry for the confusion - this is indeed the primary repository for the CF Checker.  I created the other git repository when the CF website went down and then Stephen and I decided that cedadev would be a better place to put it.  I'll try and get around to closing it soon.

Thanks for raising this problem.  If you would like to submmit a PR that would be very very useful and I will then run it through all the test files I have here and get it into the trunk asap.  There is a tarball in the repository of test files and the expected output - it is rather clunky - something else I need to update to using unit test modules.  Otherwise I will try and get around to this in the next few weeks.

Cheers,
Ros
